lib/SILGen/SILGenExpr.cpp

```diff
@@ -1894,12 +1908,25 @@ RValue RValueEmitter::visitEnumIsCaseExpr(EnumIsCaseExpr *E,
                                       {{E->getEnumElement(), t}});
   }
   
-  // Call the _getBool library intrinsic.
+  // Call the Bool(_builtinBooleanLiteral:) initializer
+  DeclName name(ctx, DeclBaseName::createConstructor(),
+                { ctx.Id_builtinBooleanLiteral });
+  auto members = ctx.getBoolDecl()->lookupDirect(name);
+  
+  // Ensure we just have one initializer
+  assert(members.size() == 1);
+  
+  // Ensure it is actually an initializer
+  assert(::isa<ConstructorDecl>(members[0]));
```

Comment:
Is the :: necessary?

Reply:
Yea the :: was necessary because of the `SILValue isa = ...`. I can just remove this though.